mdoc/Mono.Documentation/Updater/Formatters/AttributeFormatters/AttributeValueFormatter.cs

```diff
@@ -1,15 +1,20 @@
-﻿using System;
+﻿using Mono.Cecil;
+using Mono.Documentation.Util;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using Mono.Cecil;
-
-using Mono.Documentation.Util;
 
 namespace Mono.Documentation.Updater
 {
     /// <summary>Formats attribute values. Should return true if it is able to format the value.</summary>
-    class AttributeValueFormatter
+    public class AttributeValueFormatter
     {
+        private readonly string[] numbersTypeFullNames =
+        { 
+            "System.Int32",
+            "System.Int64" 
```

Comment:
🤔 there's a few more number types than this, for example: Int16 (short), Single (float), Double, and Decimal

Reply:
We don't need those numeric types, they just for tests only. I will remove them later.